ods_ci/tests/Tests/0100__platform/0101__deploy/0104__operators/0104__rhods_operator/0114__component_images.robot

```diff
@@ -2,28 +2,43 @@
 Documentation       Test Suite to check for components
 
 Resource            ../../../../../../tasks/Resources/RHODS_OLM/install/oc_install.robot
-
+Resource            ../../../../../../tests/Resources/Page/Components/Components.resource
 Suite Setup         Suite Setup
 Suite Teardown      Suite Teardown
 
 
+*** Variables ***
+${APPLICATIONS_NS}                              ${APPLICATIONS_NAMESPACE}
+${KUEUE_LABEL_SELECTOR}                         app.kubernetes.io/name=kueue
+${CODEFLARE_LABEL_SELECTOR}                     app.kubernetes.io/name=codeflare-operator
+${RAY_LABEL_SELECTOR}                           app.kubernetes.io/name=kuberay
+${TRAINING_LABEL_SELECTOR}                      app.kubernetes.io/name=training-operator
+${DATASCIENCEPIPELINES_LABEL_SELECTOR}          app.kubernetes.io/name=data-science-pipelines-operator
+${MODELMESH_CONTROLLER_LABEL_SELECTOR}          app.kubernetes.io/instance=modelmesh-controller
+${ODH_MODEL_CONTROLLER_LABEL_SELECTOR}          app=odh-model-controller
+${MODELREGISTRY_CONTROLLER_LABEL_SELECTOR}      control-plane=model-registry-operator
+${KSERVE_CONTROLLER_MANAGER_LABEL_SELECTOR}     control-plane=kserve-controller-manager
+${TRUSTYAI_CONTROLLER_MANAGER_LABEL_SELECTOR}   app.kubernetes.io/part-of=trustyai
+${WORKBENCHES_LABEL_SELECTOR}                   app.kubernetes.io/part-of=workbenches
+
+
 *** Test Cases ***
 Check For Correct Component Images
```

Comment:
As now we are waiting a few minutes for each component, could you add a [test timeout](https://robotframework.org/robotframework/latest/RobotFrameworkUserGuide.html#timeouts) to prevent waiting too much?  

Example  (not sure what will be the appropriate timeout for this test)
`[Timeout]    15`

Reply:
well, as we keep adding more components, it becomes more slowly and sometimes prone to errors, if we limit the Test Case to a fixed Timeout it is going to fail as a false positive IMO